docs/source/admin/traffic_ops.rst

```diff
@@ -324,7 +324,7 @@ This file deals with the configuration parameters of running Traffic Ops itself.
 	:workers:           This is used only by the `Legacy Perl Script`_, and sets the number of concurrent HTTP server workers allowed.
 
 :inactivity_timeout: This was used by the `Legacy Perl Script`_ to set timeouts on idle client connections to Traffic Ops - the exact operation (and even units) of this configuration option is unknown. `traffic_ops_golang`_ ignores this field.
-:influx_db_conf_path: An optional field which gives `traffic_ops_golang`_ the absolute or relative path to an `influxdb.conf`_ file. Default if not specified is a file named ``influxdb.conf`` in the same directory as this ``cdn.conf`` file.
+:influxdb_conf_path: An optional field which gives `traffic_ops_golang`_ the absolute or relative path to an `influxdb.conf`_ file. Default if not specified is to first check if the :envvar:`MOJO_MODE` environment variable is set. If it is, then Traffic Ops will look in the current working directory for a subdirectory named ``conf/``, then inside that for a subdirectory with the name that is the value of the :envvar:`MOJO_MODE` variable, and inside that directory for a file named ``influxdb.conf``. If :envvar:`MOJO_MODE` is *not* set, then Traffic Ops will look for a file named ``influxdb.conf`` in the same directory as this ``cdn.conf`` file.
```

Comment:
This reminds me, in Perl if `MOJO_MODE` was unset, it would assume `MOJO_MODE=development`.

Reply:
Should we emulate that? In 4.1.0 TO will look default to that last behavior by checking the `cdn.conf` directory. Using an implicit `MOJO_MODE` when it's not set effectively walks that back - and might make it harder to break away from `MOJO_MODE` in the future, if only slightly.

Reply:
I would if it were me, since that was the pre-existing behavior, but I don't feel too strongly about changing the default location.

Reply:
after taking, apparently, nearly a week to think about it, I think I'd like to leave the logic as-is. Because hopefully we're looking at excising the Perl entirely and it feels weird to have remnants like this still hanging around.